### PR TITLE
EF Core JSON column sample

### PR DESCRIPTION
**Pull request by alexkeh**

### CODE REVIEW

**Review by christianshay — APPROVED** [review]

File: samples/README.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ Entity Framework Core
 * Autonomous Database Sample: Demonstrates Oracle EF Core connecting to Oracle Autonomous Database. <br>
 * Dependency Injection Sample: Demonstrates using dependency injection and ASP.NET Core with Oracle EF Core. <br>
 * Getting Started Sample: Demonstrates a basic Oracle EF Core scenario using migrations and scaffolding. <br>
+* JSON Columns Sample: Demonstrates how to create an owned entity, insert, queriy, updates and delete JSON column data. <br>
```
Comment (sungam3r) [inline samples/README.md:110]: insert, queriy, updates -> insert, query, update

File: samples/ef-core/json/json-columns.cs [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        public class AuthorContext : DbContext
+        {
+            public DbSet<Author> Authors { get; set; }
+
+            //To use Oracle database JSON columns, connect to Oracle Database 21c or higher version and
+            // specify OracleSQLCompatibility to DatabaseVersion21 or higher.
+            protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+            {
+                optionsBuilder.UseOracle("User Id=<USER>; Password=<PASSWORD>; Data Source=<DATA SOURCE>"
+                    , b => b.UseOracleSQLCompatibility(OracleSQLCompatibility.DatabaseVersion21)); ;
```
Comment (sungam3r) [inline samples/ef-core/json/json-columns.cs:52]: doubled ;

File: samples/ef-core/json/json-columns.cs [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+                context.Database.EnsureDeleted();
+                context.Database.EnsureCreated();
+          
+                var author1 = new Author()
+                {
+                    Name = "John Smith",
+                    Contact = new ContactDetails()
+                    {
+                    Phone = "555 123 4567",
+                    Address = new Address("1 Any Street", "Austin", "78741", "US")
```
Comment (sungam3r) [inline samples/ef-core/json/json-columns.cs:80]: indentation

File: samples/README.md (hunk 1 above)
Comment (alexkeh) [reply samples/README.md:110]: Thanks for the review! I saw this error yesterday and fixed it already.

File: samples/ef-core/json/json-columns.cs (hunk 2 above)
Comment (alexkeh) [reply samples/ef-core/json/json-columns.cs:52]: Good catch! Filed a PR to fix this issue: https://github.com/oracle/dotnet-db-samples/pull/343

File: samples/ef-core/json/json-columns.cs (hunk 3 above)
Comment (alexkeh) [reply samples/ef-core/json/json-columns.cs:80]: Good catch! The same PR will fix this issue: https://github.com/oracle/dotnet-db-samples/pull/343

**Pull request merged**